src/http-driver.js

```diff
@@ -103,14 +108,12 @@ function makeHTTPDriver({eager = false} = {eager: false}) {
       .map(reqOptions => {
         let response$ = createResponse$(reqOptions)
         if (eager || reqOptions.eager) {
-          response$ = response$.replay(null, 1)
-          response$.connect()
+          response$ = response$.publishReplay(null, 1).connect()
+          return response$
         }
         response$.request = reqOptions
         return response$
-      })
-      .replay(null, 1)
-    response$$.connect()
+      }).shareReplay(null, 1)
```

Comment:
Can you send a commit using `publishReplay(1)` + connect()? I want to check why they are failing. We can figure out the tests later on with another PR.